<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[@Test annotaions start execution!!]]>
    </line>
    <line>
      <![CDATA[Test is Start!!!]]>
    </line>
    <line>
      <![CDATA[This is failed MethodloginPageMethodScreenShot is taken]]>
    </line>
    <line>
      <![CDATA[Test is Start!!!]]>
    </line>
  </reporter-output>
  <suite started-at="2023-04-01T19:37:57 IST" name="Default suite" finished-at="2023-04-01T19:38:04 IST" duration-ms="6182">
    <groups>
    </groups>
    <test started-at="2023-04-01T19:37:57 IST" name="Default test" finished-at="2023-04-01T19:38:04 IST" duration-ms="6182">
      <class name="testNGdynamicScreenshot.ActiTimeLogin">
        <test-method is-config="true" signature="setUp()[pri:0, instance:testNGdynamicScreenshot.ActiTimeLogin@29e495ff]" started-at="2023-04-01T19:37:57 IST" name="setUp" finished-at="2023-04-01T19:37:59 IST" duration-ms="1719" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="loginPageMethod()[pri:0, instance:testNGdynamicScreenshot.ActiTimeLogin@29e495ff]" started-at="2023-04-01T19:37:59 IST" name="loginPageMethod" finished-at="2023-04-01T19:37:59 IST" duration-ms="210" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [loginID] but found [actiTIME - Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [loginID] but found [actiTIME - Login]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at testNGdynamicScreenshot.ActiTimeLogin.loginPageMethod(ActiTimeLogin.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Test is Start!!!]]>
            </line>
            <line>
              <![CDATA[This is failed MethodloginPageMethodScreenShot is taken]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginPageMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGdynamicScreenshot.ActiTimeLogin@29e495ff]" started-at="2023-04-01T19:38:00 IST" name="tearDown" finished-at="2023-04-01T19:38:00 IST" duration-ms="183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:testNGdynamicScreenshot.ActiTimeLogin@29e495ff]" started-at="2023-04-01T19:38:00 IST" name="setUp" finished-at="2023-04-01T19:38:01 IST" duration-ms="1083" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="homePageMethod()[pri:0, instance:testNGdynamicScreenshot.ActiTimeLogin@29e495ff]" started-at="2023-04-01T19:38:01 IST" name="homePageMethod" finished-at="2023-04-01T19:38:03 IST" depends-on-methods="testNGdynamicScreenshot.ActiTimeLogin.loginPageMethod" duration-ms="2293" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Test is Start!!!]]>
            </line>
          </reporter-output>
        </test-method> <!-- homePageMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:testNGdynamicScreenshot.ActiTimeLogin@29e495ff]" started-at="2023-04-01T19:38:03 IST" name="tearDown" finished-at="2023-04-01T19:38:04 IST" duration-ms="183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testNGdynamicScreenshot.ActiTimeLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
